spec/rails_helper.rb

```diff
@@ -30,6 +30,7 @@
 
 RSpec.configure do |config|
   config.include ActiveJob::TestHelper
+  config.include Devise::Test::ControllerHelpers, type: :controller
```

Comment:
Deze zorgt neem ik aan voor de `sign_in` helper?

Reply:
Dat weet ik niet zeker, maar hij zorgt er in elk geval voor dat de devise middlewares worden ingeladen zodat dit ook gebruikt worden (kwam uit een error melding van Devise dat je dit moest doen)